Solution_continousArray.java

```diff
@@ -0,0 +1,36 @@
+import java.util.*;
```

Comment:
Please include: 
```
// Time Complexity :
// Space Complexity :
// Did this code successfully run on Leetcode :
// Any problem you faced while coding this :


// Your code here along with comments explaining your approach
```

Reply:
Thank you for the feedback, Sure I will make sure from the next problem I will add all details.